JSDemos/Demos/DataGrid/StatePersistence/description.md

```diff
@@ -1 +1,5 @@
-Sort and filter data, group, reorder and resize columns, change page numbers and page size. Once you are done, refresh the web page to&nbsp;see that the grid&rsquo;s state is&nbsp;automatically persisted to&nbsp;continue working from where you stopped.
\ No newline at end of file
+**DataGrid** state can be persistent - if a user sorts and filters data, groups, reorders and resizes columns or makes other changes, then the widget saves these modifications and restores them when a user reloads the page. The full list of modifications that can be saved is in the [stateStoring](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/stateStoring/) documentation.
```

Comment:
- "... or makes other changes..." - Oxford comma missing
- "... then the widget saves these modifications..." - "then" associates with time or order. Neither is the case.
- "The full list of modifications **that can be saved**" - unnecessary adjective clause
- "The full list of modifications ... is in the stateStoring documentation" - "is" is a weak verb. Actually, I've rearranged the last sentence because, to my mind, imperative mood fits better here.

```suggestion
**DataGrid** state can be persistent - if a user sorts and filters data, groups, reorders and resizes columns, or makes other changes, the widget saves these modifications and restores them when a user reloads the page. Refer to the [stateStoring](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/stateStoring/) article for a full list of saved modifications.
```

Reply:
@RomanTsukanov 
* "... then the widget saves these modifications..." - "then" grammatically fits into this sentence (see https://forum.thefreedictionary.com/postst156297_if---then.aspx). This word can be ommited though, but I used it to stress the beginning of the main clause. Should it be here?

* "saved modifications" -> "savable modifications": "savable" suits better, doesn't it?

* Is stateStoring an article or a documentation?

Reply:
- I see why you used "then", but it is unnecessary.
- "Savable" is a pretty rare word, better play safe. Let's return the adjective clause.
- Documentation is a whole piece. It consists of various articles: tutorials, guides, etc.

```suggestion
**DataGrid** state can be persistent - if a user sorts and filters data, groups, reorders and resizes columns, or makes other changes, the widget saves these modifications and restores them when a user reloads the page. Refer to the [stateStoring](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/stateStoring/) article for a full list of modifications that can be saved.
```